src/CommandLine/TopicPerEventTopologyEndpoint.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace NServiceBus.Transport.AzureServiceBus.CommandLine;
+
+using System;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus;
+using Azure.Messaging.ServiceBus.Administration;
+using McMaster.Extensions.CommandLineUtils;
+
+static class TopicPerEventTopologyEndpoint
+{
+    public static async Task Create(ServiceBusAdministrationClient client, CommandArgument name, CommandOption<int> size, CommandOption partitioning)
+    {
+        try
+        {
+            await Queue.Create(client, name, size, partitioning);
+        }
+        catch (ServiceBusException ex) when (ex.Reason == ServiceBusFailureReason.MessagingEntityAlreadyExists)
+        {
+            Console.WriteLine($"Queue '{name.Value}' already exists, skipping creation");
```

Comment:
Did we use this before? Would it be better to use Console.Error?

Reply:
I've checked and this is how it worked. Moreover, these are not really error messages because the commands complete successfully so I'll leave it that way.